### ISSUE TITLE
Implement moxie combat

### ISSUE THREAD

**Issue by sadovsf** [sadovsf/StargateRPGSystem#2]
Moxie does not work currently and have to be replaced by some form of user self tracking or some tricky combat systems bending

**Comment by budbreaker** [comment]
Hi, how do you propose moxie to work? I have accomplished rolling with moxie bonus (entered by hand in character sheet) similar to the initiative roll. 

Using the higher stat automatically seems to be bigger problem if even possible since group rolls are implemented together with single rolls in Foundry core.
![image](https://user-images.githubusercontent.com/64734725/150343149-eaeb811c-eb4e-433b-9507-3ced7fa47fa4.png)

**Comment by sadovsf** [comment]
Thats awesome! Main thing is that it should work as it is described in rules where i believe people then go in opposite order than what they throw. Then there is determination chips system which would be good to implement in some way as well. 

Other than this it should be fairly straight forward 🤔

**Comment by budbreaker** [comment]
Looks like the sorting stuff is tied to the combat tracker itself which means that in order to do that the combat tracker will need to be extended in the core or a new one implemented for the module. One (lame) workaround would be to use inverse moxie modifier and aim to roll the lowest number ¯\_(ツ)_/¯

**Comment by sadovsf** [comment]
True, but that would be quite confusing for players i think. They would roll something else than what they get, plus various possible advantages and other calculations and bonuses would break it quite easily. Over all its nut much future proof and could be really hard to support everywhere in long run :-(

**Comment by budbreaker** [comment]
Or maybe the formula could be like `50(or so) - rollWithBonuses`. That way the player with highest roll would have the lowest number.

**Comment by sadovsf** [comment]
Sure It would work now but such solutions almost always eventually bite you in the ass sooner or later 😁 (speaking from 10y in game/engine development experiences)

**Comment by budbreaker** [comment]
Tak nič :D

**Comment by sadovsf** [comment]
Opravdu to chce nějaké koncepcnejsi řešení 😉

(it really needs some more conceptual solution)
